index.html

```diff
@@ -102,16 +118,20 @@ <h1 class="categories">Welcome to my portfolio!</h1>
       by Marshall Miller
     </footer>
 
-    <script src="vendors/js/page.js"></script>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/handlebars.js/4.0.5/handlebars.js"></script>
+    <script src="vendors/js/jquery.bxslider.js"></script>
+    <script src="vendors/js/page.js"></script>
+    <script src="js/portfolio.js"></script>
+    <script src="js/tabSlide.js"></script>
     <script src="js/aboutController.js"></script>
     <script src="js/schoolController.js"></script>
     <script src="js/workController.js"></script>
+    <script src="js/githubToken.js"></script>
+    <script src="js/repoView.js"></script>
+    <script src="js/repo.js"></script>
     <script src="js/projectController.js"></script>
     <script src="js/routes.js"></script>
-    <script src="js/tabSlide.js"></script>
-    <script src="js/portfolio.js"></script>
     <script>
```

Comment:
We still want to get these calls out of the the index. Is there a better place to put them?